tools/src/main/scala/caliban/tools/SchemaWriter.scala

```diff
@@ -298,24 +298,24 @@ object SchemaWriter {
          |}""".stripMargin
     }
 
-    def replaceNameOfInnertype(name: String, ftype: Type): Type =
+    def replaceNameOfInnerType(name: String, ftype: Type): Type =
```

Comment:
It's possible `Innertype` is a reserved concept, but I'm hoping it's just `Inner`+`Type`